gallery/how-to/managing-profile/creating-APIkeys.md

```diff
@@ -1,73 +1,90 @@
+Scoped API keys can assign any of the following permissions:
 
-* Push new packages
-* Push new or update packages
-* Unlist packages
+- Push new packages
+- Push new or update packages
+- Unlist packages
 
-In addition, any key you create will have an expiration. The default is a year, and may be set for 1, 90, 180, 270, or 365 days. 
+Every new key has an expiration. The expiration value is measured in days. The possible values for expiration are:
 
-Note that these settings cannot be changed once the key is created. 
+- 1 day
+- 90 days
+- 180 days
+- 270 days
+- 365 days (default)
 
-## Editing and Deleting Existing API Keys
+These settings cannot be changed once the key is created. You cannot create a new key that never
+expires.
 
-Once an API key is created, you may update the set of packages controlled by existing keys,  regenerate the key value, and delete the key. 
-As noted above, you cannot modify the security scope for an existing API key, or how long the key may be used. 
-The options available are shown in the screen shot below:
+## Editing and deleting existing API keys
 
-![Obtaining new API key value](../../Images/PSGallery_EditAPIKey.png)
+You can change some settings of an existing key. As previously noted, you cannot modify the
+security scope for an existing API key or change the expiration. The changeable options are shown in
+the following screenshot:
 
-To change the packages controlled by a key, you can choose individual packages from the list, or change the glob pattern as needed. 
+![Obtaining new API key value](../../Images/PSGallery_EditAPIKey.png)
 
-Use Regenerate to refresh the key value. 
-Just like when you initially create a key, you must Copy the key value immediately after updating it, as the Copy option will not be available once the page is refreshed. 
+To change the packages controlled by a key, you can choose individual packages from the list or
+change the glob pattern.
 
-You may Delete an existing key by choosing Delete, and then confirming the action in the pop-up that will be displayed. 
-Once a key is deleted, it will be unusable. 
+Clicking **Regenerate** creates a new key value. Just like when you initially created the key, you
+must **Copy** the key value immediately after updating it. The **Copy** option is not available
+once you leave this page.
 
-## Key Expiration
+Clicking **Delete** displays a confirmation message. Once a key is deleted, it will be unusable.
 
-A warning e-mail will be sent to the account holder 10 days before the expiration of an API key so that you can act on it well in advance. 
+## Key expiration
 
-If any key expires, a warning message will be displayed at the top of the API key management page, showing which key is no longer valid. 
+Ten days before the expiration, the PowerShell Gallery sends a warning e-mail the account holder of
+the API key. You must generate a new key before the expiration date.
```

Comment:
The "you must" isn't accurate. Key part of the message:
> Key will become unusable
> You can visit the site, regenerate, and then it will be usable.

Reply:
fixed